cli/src/pcluster/api/encoder.py

```diff
@@ -9,15 +9,16 @@
 # Generated by OpenAPI Generator (python-flask)
 
 import datetime
+import json
 
 import six
-from connexion.apps.flask_app import FlaskJSONEncoder
+from flask.json.provider import DefaultJSONProvider
 
 from pcluster.api.models.base_model_ import Model
 from pcluster.utils import to_iso_timestr
 
 
-class JSONEncoder(FlaskJSONEncoder):
+class JSONEncoderForCli(json.JSONEncoder):
```

Comment:
Why `ForCli`? This encoder is used also by the API. Why not keeping the original class name?